api-reference/50 Common/utils/pdfExporter/exportDataGrid(options).md

```diff
@@ -4,13 +4,228 @@ module: pdf_exporter
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Exports grid data to a PDF file.
 
 ##### return: Promise<void>
-<!-- Description goes here -->
+A <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise" target="_blank">Promise</a> that is resolved with an object that contains the coordinates of the first and last cells.
 
 ##### param(options): PdfExportDataGridProps
-<!-- Description goes here -->
+Export settings.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+This method requires the <a href="https://github.com/MrRio/jsPDF" target="_blank">jsPDF</a> library to export data and the <a href="https://github.com/simonbengtsson/jsPDF-AutoTable" target="_blank">jsPDF-AutoTable</a> plugin to create tables in exported files.
+
+You can call this method at any point in your application.
+
+---
+##### jQuery
+
+    <!--JavaScript-->
+    $(function(){
+        $('#exportButton').dxButton({
+            // ...
+            onClick: function() {
+                const doc = new jsPDF();
+                DevExpress.pdfExporter.exportDataGrid({
+                    jsPDFDocument: doc,
+                    component: dataGrid
+                }).then(function() {
+                    doc.save('Customers.pdf');
+                });
+            }
+        });
+
+        let dataGrid = $('#gridContainer').dxDataGrid({
+            // Specify grid options here
+        }).dxDataGrid('instance');
+    });
+
+
+    <!--HTML-->
+    <head>
+        <!-- ... -->
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/jspdf/2.0.0/jspdf.umd.min.js"></script>
+        <script src=""https://cdnjs.cloudflare.com/ajax/libs/jspdf-autotable/3.5.9/jspdf.plugin.autotable.min.js"></script>
+        <!-- reference the DevExtreme sources here -->
+    </head>
+
+##### Angular   
+
+    <!-- tab: Installation command -->
+    npm install jspdf jspdf-autotable
+
+    <!-- tab: app.component.html -->
+    <dx-button ... 
+        (onClick)="exportGrid($event)">
+    </dx-button>
+
+    <dx-data-grid ...
+        id='gridContainer'
+        ref={this.dataGridRef}
+        >    
+        <!-- Specify grid options here -->
+    </dx-data-grid>
+
+    <!-- tab: app.component.ts -->
+    import { Component } from '@angular/core';
+    import Exporter from 'devextreme/pdf_exporter';
```

Comment:
We can use the named import for this purpose as follows:
import { exporDataGrid } from 'devextreme/pdf_exporter'
Don't we need use aliase for this code snippets?
import { exporDataGrid as exportDataGridToPdf } from 'devextreme/pdf_exporter'